code/game/machinery/computer/arcade/arcade.dm

```diff
@@ -556,10 +556,11 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 			playsound(loc, 'sound/arcade/win.ogg', 50, TRUE)
 
 			if(obj_flags & EMAGGED)
-				new /obj/effect/spawner/newbomb/plasma(loc, /obj/item/assembly/timer)
+				// new /obj/effect/spawner/newbomb/plasma(loc, /obj/item/assembly/timer) - ORIGINAL
+				new /obj/item/gun/energy/recharge/ebow(loc) // SKYRAT EDIT CHANGE - Weapon instead of bomb
```

Comment:
Swap it for a minibomb instead, it's far more fitting for winning against this arcade.